final-project/manProtocol.tex

```diff
@@ -88,4 +88,10 @@ \subsection{Messages sent during Join and Leave}
 	}
 \end{verbatim}
 
-These messages conform to the the protocol defined by the types in the \href{https://github.com/dyc3/opentogethertube/blob/master/common/models/messages.ts}{OTT monolith here}.
\ No newline at end of file
+These messages conform to the the protocol defined by the types in the \href{https://github.com/dyc3/opentogethertube/blob/master/common/models/messages.ts}{OTT monolith here}.
+
+\section{Messages sent during Monolith connection startup}
+
+When a Monolith starts up, and it has balancers configured, it must initiate a websocket connection to each of the balancers. The Monolith must send an "init" message to the Balancer to inform it of the port that it is listening for HTTP requests on, and an auth token to verify authenticity.
+
+Once the Balancer receives this message, the connection is considered established, and the Monolith and Balancer can begin sending messages to each other.
```

Comment:
Just curious, is this the point where a node would be added to the record of states on the balancer? or is that not until specific room instances are loaded.

Reply:
The Balancer does not know about rooms until the Monoliths tell them via gossip. Before the init message, it is invalid behavior to send other protocol messages. After the init message, it becomes valid behavior for both ends of the connection to start sending protocol messages (gossip, client messages, join/leave, etc)